### PR TITLE
CI/CD pipeline example for Jenkins with Apigee X/hybrid

### PR DESCRIPTION
**Pull request by danistrebel**
What's changed, or what was fixed?

- Expanding Jenkinsfile to work with X/hybrid
- Adding X/hybrid to jenkinsfilerunner in pipeline.sh
- Documenting how to configure GCE deployment for running jenkins

- [x] I have run all the tests locally and they all pass.
- [x] I have followed the relevant style guide for my changes.

**CC:** @apigee-devrel-reviewers

### CODE REVIEW

File: references/cicd-pipeline/jenkins-build/README.md [hunk 1]
```diff
@@ -19,52 +19,158 @@ instance.
...
+  --member="serviceAccount:$JENKINS_SA_EMAIL" \
+  --role="roles/apigee.environmentAdmin"
+
+gcloud projects add-iam-policy-binding "$PROJECT_ID" \
+  --member="serviceAccount:$JENKINS_SA_EMAIL" \
+  --role="roles/apigee.apiAdmin"
+
+gcloud projects add-iam-policy-binding "$PROJECT_ID" \
+  --member="serviceAccount:$JENKINS_SA_EMAIL" \
+  --role="roles/storage.objectViewer"
```
[the remark this answers is not in the document]
Comment (danistrebel) [reply references/cicd-pipeline/jenkins-build/README.md:78]: to authorize the VM to pull the images from GCR.

**Review by OmidTahouri — APPROVED** [review]
LGTM

**Pull request merged**